frontend/public/locales/en/gcweb.json

```diff
@@ -9,7 +9,15 @@
     "govt-of-canada.text": "Government of Canada",
     "govt-of-canada.href": "https://www.canada.ca/en.html",
     "menu-title": "Account",
-    "menu-sign-out": "Sign Out"
+    "menu-dashboard.text": "My Dashboard",
+    "menu-dashboard.href": "en/my-dashboad",
```

Comment:
I think this would be better as

``` json
"menu-dashboard.href": "{{baseUri}}/en/my-dashboad"
```

(plus see associated comment in `_gcweb-app.tsx`)

Reply:
This comment applies to all href properties in this file.